squawk/data/dataset.py

```diff
@@ -20,6 +22,33 @@ def __post_init__(self):
         self.num_labels = len(self.label_map)
 
 
+class LruCache(object):
```

Comment:
something very minor,
I prefer LRUCache as LRU itself is acronym but I am fine if you decide to keep it as it is.

Reply:
Depending on the codebase, acronyms are camelcased -- e.g., HtmlParser.